src/mongocxx/client.hpp

```diff
@@ -107,6 +108,27 @@ class MONGOCXX_API client {
     MONGOCXX_DEPRECATED void* implementation() const;
 
     ///
+    /// Sets the read concern for this client.
+    ///
+    /// Modifications at this level do not effect existing databases instances that have have been
+    /// created by this client but do effect new ones as databases inherit the @c read_concern
+    /// settings of their parent upon instantiation.
+    ///
+    /// @param rc
+    ///   The new @c read_concern
+    ///
+    /// TODO add link to docs once they exist.
+    ///
+    void read_concern(class read_concern rc);
+
+    ///
+    /// Returns the current read concern for this client.
+    ///
+    /// @return The current @c read_concern
+    ///
+    class read_concern read_concern() const;
```

Comment:
@Machyne Ping? This doesn't seem to be done, though you had a comment that indicated that it was. Do you need to push a commit?